src/app.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = () => {
+  try {
+    if (process.argv[2] === 'cp') {
+      const [file, copy] = process.argv.slice(3);
+
+      fs.copyFileSync(file, copy);
```

Comment:
Add a condition to check if 'file' and 'copy' are different. If they are the same, the function should not copy the file.